app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ListsController < ApplicationController
+  def new
+    @list = board.lists.build
+  end
+
+  def create
+    @list = board.lists.build(list_params)
```

Comment:
i tak i nie ;) wlasnie to niby sie nie różni ale w niektórych relacjach trzeba uzywac build ;) to nie do konca jest to samo chociaz wiem ze railsy mowią ze to alias ;) Ogólnie new chyba jest nowsze wiesz uzywaj `new` kiedys spotkasz sie z przpadkiem ze bedzie trzeba uzyc build :)